automake-1.14 freaks out if AM_PROG_AR is not defined.

--- configure.ac.orig	2014-02-17 19:47:41.572637701 +0100
+++ configure.ac	2014-02-17 19:47:54.604474781 +0100
@@ -14,6 +14,7 @@ AC_CONFIG_HEADER([src/lib/config.h])
 
 AM_INIT_AUTOMAKE([-Wall -Werror foreign])
 AM_MAINTAINER_MODE
+AM_PROG_AR
 
 AC_CANONICAL_HOST
 CANONICAL_HOST=$host
